mappings/net/minecraft/village/PointOfInterestStorage.mapping

```diff
@@ -9,6 +9,9 @@ CLASS aqa net/minecraft/village/PointOfInterestStorage
 		FIELD d predicate Ljava/util/function/Predicate;
 		METHOD a getPredicate ()Ljava/util/function/Predicate;
 	FIELD a pointOfInterestDistanceTracker Laqa$a;
+	METHOD a initForPalette (Lbgn;Lbxd;)V
+	METHOD a mayPointOfInterestAppearIn (Lbxd;)Z
+	METHOD a createPointsOfInterest (Lbxd;Lfo;Ljava/util/function/BiConsumer;)V
```

Comment:
I think these shouldn't have `PointOfInterest` in the names to match the other methods (`add`, `remove`, `getDistanceFromNearestOccupied`, `count`). 

Also, what does this method do differently from `add`?

Reply:
this one creates poises if they aren't present; then it adds poi for each matching state in the chunk